src/logic/calculate.js

```diff
@@ -0,0 +1,13 @@
+export default function calculate(calculatorData, buttonName) {
+  let { total, next } = calculatorData;
+  const { operation } = calculatorData;
+
+  if (buttonName === '+/-') {
+    total *= -1;
+    next *= -1;
+  }
+  if (next && buttonName === '%') next /= 100;
+  if (total && buttonName === '%') total /= 100;
```

Comment:
According to the project requirement, we need to make use of the module;
![image](https://user-images.githubusercontent.com/31897434/100809444-6b6b3b00-3436-11eb-9612-673806acef9c.png)

Kindly add the module from Big.js in `src/logic/operate.js` instead of hard-coding.

### Heads-Up:
When it comes to `%` we only need one operand.
example: click `10` then `%` = 0.01